pkg/env/prod.go

```diff
@@ -167,7 +168,7 @@ func newProd(ctx context.Context, log *logrus.Entry) (*prod, error) {
 		}
 		p.acrDomain = acrResource.ResourceName + "." + p.Environment().ContainerRegistryDNSSuffix
 	} else {
-		p.acrDomain = "arointsvc" + "." + azure.PublicCloud.ContainerRegistryDNSSuffix // TODO: make cloud aware once this is set up for US Gov Cloud
+		p.acrDomain = "arointsvc" + "." + azureclient.PublicCloud.ContainerRegistryDNSSuffix // TODO: make cloud aware once this is set up for US Gov Cloud
```

Comment:
Comment is not needed anymore?

Reply:
The comment is still needed, because it is still hardcoded to public cloud. This change is just to be consistent with the rest of the codebase, but once we can switch over to use the FF ACR, we will.